### PR TITLE
Attempt at heartbeat pre-merging

### PR DESCRIPTION
**Pull request by johan-bjareholt**

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
Squash these commits when done

File: aw_client/client.py [hunk 1]
```diff
@@ -149,18 +154,33 @@ def get_eventcount(self, bucket_id: str, limit: int=100, start: datetime=None, e
...
 
-    def heartbeat(self, bucket_id: str, event: Event, pulsetime: float, queued: bool=False) -> Optional[Event]:
+    def heartbeat(self, bucket_id: str, event: Event, pulsetime: float, queued: bool=False):
         """ This endpoint can use the failed requests retry queue.
             This makes the request itself non-blocking and therefore
             the function will in that case always returns None. """
 
+        from aw_transform.heartbeats import heartbeat_merge
         endpoint = "buckets/{}/heartbeat?pulsetime={}".format(bucket_id, pulsetime)
-        data = event.to_json_dict()
```
Comment (ErikBjare) [inline aw_client/client.py]: I don't think premerging of heartbeats should be done at all if `queued=False`, would make the code clearer as well I think.

**Comment by ErikBjare** [comment]
Just saw your message in Messenger saying "this is completely broken". Please elaborate.

**Comment by johan-bjareholt** [comment]
@ErikBjare Since the "pre-merge attempt 2" commit it is not completely broken anymore!

File: aw_client/client.py (hunk 1 above)
Comment (johan-bjareholt) [reply aw_client/client.py]: True.

Something unrelated to this PR though, queue should probably be on by default. In most cases the watcher should be dumb and only send data and not really care about the response.

**Comment by johan-bjareholt** [comment]
Please review, works well

**Pull request merged**